src/xr_parser.py

```diff
@@ -146,14 +146,17 @@ def _old_axes_dict(self, var_name=None):
         coords_list = list(axes_obj.coords)
```

Comment:
Remove the changes to xr_parser.py. I believe the fix ended up being addressed elsewhere in PR #185